packages/compartment-mapper/src/parse-cjs.js

```diff
@@ -25,35 +25,44 @@ export const parseCjs = async (
    * @param {Compartment} compartment
    * @param {Record<string, string>} resolvedImports
    */
-  const execute = async (moduleExports, compartment, resolvedImports) => {
+  const execute = (moduleExports, compartment, resolvedImports) => {
     const functor = compartment.evaluate(
       `(function (require, exports, module, __filename, __dirname) { ${source} //*/\n})\n//# sourceURL=${location}`,
     );
-
+    let exportsReferenceCopy = moduleExports;
     const module = freeze({
       get exports() {
-        return moduleExports;
+        return exportsReferenceCopy;
       },
       set exports(value) {
-        moduleExports.default = value;
+        exportsReferenceCopy = value;
```

Comment:
This will not make the `module.exports=something` become available as the default to other modules synchronously. 
Doesn't seem to be how they're using it though. I could try to come up with a test that breaks it.